packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -377,18 +377,11 @@ class BottomNavigationBar extends StatefulWidget {
   /// The cursor for a mouse pointer when it enters or is hovering over the
   /// items.
   ///
-  /// If [mouseCursor] is a [WidgetStateProperty<MouseCursor>],
-  /// [WidgetStateProperty.resolve] is used for the following [WidgetState]s:
-  ///
-  ///  * [WidgetState.selected].
+  /// If [mouseCursor] is a [WidgetStateMouseCursor], then its `resolve` method
+  /// can depend on whether the [WidgetState.selected] state is active.
```

Comment:
This wording feels odd. Maybe something like this?
```suggestion
  /// If [mouseCursor] is a [WidgetStateMouseCursor], then its `resolve` method
  /// can define the behavior of the cursor depending on whether the [WidgetState.selected] state is active.
```
Or something similar to the other documentation comments.

Reply:
yes, I like your wording better 🙂👍